packages/react/src/FormErrorList/FormErrorList.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes } from 'react'
+import { Alert } from '../Alert'
+import { HeadingLevel } from '../Heading/Heading'
+import { LinkList } from '../LinkList'
+
+export type FormErrorListProps = {
+  errors: { id: string; label: string }[]
+  heading?: string
+  headingLevel?: HeadingLevel
+} & HTMLAttributes<HTMLDivElement>
+
+export const FormErrorList = forwardRef(
+  (
+    {
+      className,
+      errors,
+      heading = 'Verbeter de fouten voor u verder gaat',
+      headingLevel = 2,
+      ...restProps
+    }: FormErrorListProps,
+    ref: ForwardedRef<HTMLDivElement>,
+  ) => {
+    if (errors.length === 0) return undefined
+
+    return (
+      <Alert
+        {...restProps}
+        className={clsx('ams-form-error-list', className)}
+        headingLevel={headingLevel}
+        ref={ref}
+        role="alert"
+        severity="error"
+        title={heading}
+      >
+        <LinkList>
```

Comment:
This could be just me but I associate the chevrons with links that take you away from the page (like in the footer and related links), not anchors on the page. I would've gone with the `UnorderedList` component. And the lack of padding does not do it justice, it's way too close to the title. It needs the inside spacing here.

Reply:
Yeah, this is just the basic structure using existing components. [I've added a ticket for design to look at it.](https://gemeente-amsterdam.atlassian.net/browse/DES-792)